pkg/sac/test_scope_checker_core.go

```diff
@@ -0,0 +1,162 @@
+package sac
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/auth/permissions"
+	"github.com/stackrox/rox/pkg/sac/effectiveaccessscope"
+)
+
+// TestClusterScope contains cluster-level scope information (cluster included or list of included namespaces
+// in the cluster) to build test scope checkers.
+type TestClusterScope struct {
+	Namespaces []string
+	Included   bool
+}
+
+// TestResourceScope contains resource-level scope information (resource fully included or list of included clusters
```

Comment:
by `resource fully included` do you mean global access?

Reply:
Indeed, resource fully included means global access (that is, access to objects of the resource type in any cluster and namespace).